src/ui/include/mainwindow.h

```diff
@@ -206,10 +206,10 @@ private slots:
     DocEngine*            m_docEngine;
     QMenu*                m_tabContextMenu;
     QList<QAction *>      m_tabContextMenuActions;
-    QPushButton*          m_sbFileFormatBtn;
-    QPushButton*          m_sbEOLFormatBtn;
-    QPushButton*          m_sbTextFormatBtn;
-    QPushButton*          m_sbOvertypeBtn;
+    QPushButton* m_sbFileFormatBtn;
+    QPushButton* m_sbEOLFormatBtn;
+    QPushButton* m_sbTextFormatBtn;
+    QPushButton* m_sbOvertypeBtn;
```

Comment:
This one's another annoying bit about clang-format. It's not too bad with several variables of the same type, but if we were to remove all the extra indentation it would definitely become less readable.

Reply:
Yea.... I'm fairly fond of how we do header files so it might not be a bad idea to restrict that.